products/statement-generator/src/components/FAQAccordion.tsx

```diff
@@ -14,19 +14,20 @@ interface CustomAccordionProps {
   content?: string[];
 }
 
-const useStyles = makeStyles(({ palette }) =>
+const useStyles = makeStyles(() =>
   createStyles({
     accordionWrapper: {
       width: '100%',
-      color: palette.primary.darker,
       boxShadow: 'none',
       border: 'none',
-      position: 'static', // when position is 'relative' or not specified a vertical line appears above the accordion (unclear why)
+      position: 'static',
+      '& *': {
```

Comment:
My only suggestion is that we make this `& a` instead of `& *` just to be a little more specific.